recipes/r-cpprouting/meta.yaml

```
@@ -0,0 +1,108 @@
+{% set version = '3.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-cpprouting
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/cppRouting_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/cppRouting/cppRouting_{{ version }}.tar.gz
+  sha256: bed3a5545f4e8b2ad849735c499149a5ee668d032ec54eb9ce1e523ec53417d6
+
+build:
+  skip: True  # [win]
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+    - lib/R/library/RcppParallel/libs/  # [osx]
+    
+
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+    - '*/libtbb.dylib'   # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ stdlib("c") }}
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rcpp >=1.0.7
+    - r-rcppparallel
+    - r-rcppprogress
+    - r-data.table
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-rcpp >=1.0.7
+    - r-rcppparallel
+    - r-rcppprogress
+    - r-data.table
+
+test:
+  commands:
+    - $R -e "library('cppRouting')"           # [not win]
+    - "\"%R%\" -e \"library('cppRouting')\""  # [win]
+
+about:
+  home: https://github.com/vlarmet/cppRouting
+  license: GPL-2.0-only
```

DESCRIPTION shows `GPL (>=2)`, so

```suggestion
  license: GPL-2.0-or-later
```